### ISSUE TITLE
Auto-whitelist mtalk.google.com

### ISSUE THREAD

**Issue by ale5000-git** [AdAway/AdAway#982]
Auto-whitelist (without doing it manually) mtalk.google.com

Since blocking it break Google Cloud Messaging and there have been reports it happened in the past.
So I think it may be a good idea to hardcode this exception.

**Comment by guest4711** [comment]
No, don't agree. If a hostfile may break Google Cloud Messaging it would be fine to add an entry to the FAQ regarding it. But it would be the wrong way to hardcode a whitelist (you could not override it). It would be acceptable for me to populate a default whitelist with the appropriate settings (and to add it to FAQ). But if you begin to hard-whitelist one host (of a company living from adds and our data) you cannot argue not to whitelist another (e.g. doubleclick.net) while some apps do not function properly when these hosts are blocked....

**Comment by ale5000-git** [comment]
A default whitelist would be very good.
